### ISSUE TITLE
Update for MetaMask changes?

### ISSUE THREAD

**Issue by spoonknife22** [trufflesuite/drizzle#109]
Is there an update coming for the injection changes made by MetaMask? As of now it only works with the legacy MM extension.

**Comment by thisisandreeeee** [comment]
I share similar concerns as to how drizzle will comply with [EIP-1102](https://eips.ethereum.org/EIPS/eip-1102). Is drizzle flexible enough to handle a flow where users click a button, a metamask notification pops up, and they can select their desired accounts? Or would you recommend that we use react for this instead?

**Comment by eggplantzzz** [comment]
Hello! We are currently working on getting a plan together for updating Drizzle. Hang tight and sorry for the delay!

**Comment by spoonknife22** [comment]
> Hello! We are currently working on getting a plan together for updating Drizzle. Hang tight and sorry for the delay!

Glad to hear it! Thanks!

**Comment by eggplantzzz** [comment]
Hey again @isaiahcurry8 and @thisisandreeeee, we had a discussion about updating the library and it looks like Drizzle might be getting deprecated for the time being. We had planned on updating it but there are too many things that need attention right now. Sorry to keep you hanging for so long.

**Comment by spoonknife22** [comment]
That is awful news. I have bee building on Drizzle for a year now. Very
disappointing.

**Comment by rsodre** [comment]
> Hey again @isaiahcurry8 and @thisisandreeeee, we had a discussion about updating the library and it looks like Drizzle might be getting deprecated for the time being. We had planned on updating it but there are too many things that need attention right now. Sorry to keep you hanging for so long.

And I'm releasing a project this week!
What does 'for a while' mean exactly?
Is it going to be updated eventually or probably not anymore...

Where can I get more info about the MettaMask updates?
Any tips about another good store that works with truffle?

**Comment by eggplantzzz** [comment]
Yes, sorry about this. It is possible that at some point in the future we will pick this project back up but as of now we don't have time to maintain it. If you are handy with JS feel free to clone it and update it yourself. Sorry again!

After a quick search I found [this article](https://medium.com/metamask/breaking-changes-to-the-metamask-provider-are-here-7b11c9388be9) about the changes.
